src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -745,6 +747,15 @@ private void AddInfo(string message)
             }
         }
 
+        private void AddError(string message)
+        {
+            lock (this)
```

Comment:
There is no need for the lock here.
The reason that we added the lock in `AddInfo` is that there could be concurrent writers (concurrent writes to the cloud).
At this point, there is only a single thread that can write the errors.

Reply:
what about reads?

Reply:
changed List to ConcurrentQueue